src/org/zalando/stups/essentials/external/kio.clj

```diff
@@ -0,0 +1,20 @@
+(ns org.zalando.stups.essentials.external.kio
+  (:require [clj-http.client :as http]
+            [org.zalando.stups.friboo.log :as log]
+            [org.zalando.stups.friboo.system.oauth2 :as oauth]))
+
+(defn- make-options
+  [tokens]
+  {:headers {:authorization (str "Bearer " (oauth/access-token :kio tokens))}
+   :accept :json
+   :as :json})
+
+(defn get-app
```

Comment:
It would be nice to have this operation wrapped into a Hystrix command `(defcommand ...` instead of `(defn ...`